startform/src/main/kotlin/nl/nlportal/startform/autoconfigure/StartFormAutoConfiguration.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.nlportal.startform.autoconfigure
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import nl.nlportal.data.liquibase.LiquibaseMasterChangeLogLocation
+import nl.nlportal.startform.autodeployment.StartFormApplicationReadyEventListener
+import nl.nlportal.startform.autodeployment.StartFormDeploymentService
+import nl.nlportal.startform.repository.StartFormRepository
+import nl.nlportal.startform.service.StartFormService
+import nl.nlportal.zgw.objectenapi.service.ObjectenApiService
+import org.springframework.boot.autoconfigure.AutoConfiguration
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean
+import org.springframework.boot.autoconfigure.domain.EntityScan
+import org.springframework.boot.context.properties.EnableConfigurationProperties
+import org.springframework.context.annotation.Bean
+import org.springframework.core.io.ResourceLoader
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories
+
+@AutoConfiguration
+@EnableJpaRepositories(basePackages = ["nl.nlportal.startform.repository"])
+@EntityScan("nl.nlportal.startform.domain")
+@EnableConfigurationProperties(StartFormConfig::class)
+class StartFormAutoConfiguration {
+    @Bean
+    @ConditionalOnMissingBean(StartFormService::class)
```

Comment:
Is dit wel nodig, kan er een al een andere Bean bestaan van dit type??

Reply:
Dit word op heel veel andere plekken ook gedaan in het project en vanuit Valtimo is dit de standaart manier van doen deze heb ik gewoon overgenomen